assets/img/sdw-write.svg

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 23.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 150 150" style="enable-background:new 0 0 150 150;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#FC4F08;}
+	.st1{fill:none;stroke:#FC4F08;stroke-width:6;stroke-miterlimit:10;}
+	.st2{fill-rule:evenodd;clip-rule:evenodd;fill:#FC4F08;}
+	.st3{fill-rule:evenodd;clip-rule:evenodd;fill:#FDC9B6;}
+	.st4{fill:#FDC9B6;stroke:#FC4F08;stroke-width:3.5;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;}
+	.st5{fill:#FFFFFF;}
+	.st6{fill:#60B530;}
+	.st7{fill:none;stroke:#60B530;stroke-width:5.1121;stroke-miterlimit:10;}
+	.st8{fill:none;}
+	.st9{font-family:'MyriadPro-Regular';}
```

Comment:
The `MyriadPro-Regular` font is not installed by default in browsers. Browsers will use a default font instead. In my case, that's `Times New Roman` which is far from ideal to render `{ ; }` and which breaks the alignment. I would suggest switching to `font-family: Sans-Serif;`, which should yield a good result across platforms.

Note the text also needs to be moved a bit, see below :)

Reply:
The artist made me a new version using a vector for the text instead of a font, is that okay too?

Reply:
Sure!

Reply:
I added the new version, can you recheck?